app.js

```diff
@@ -0,0 +1,98 @@
+const express = require("express");
+require("express-async-errors");
+
+const app = express();
+
+app.set("view engine", "ejs");
+app.use(require("body-parser").urlencoded({ extended: true }));
```

Comment:
You could use `express.urlencoded` rather than `body-parser's` `urlencoded` here. I believe they are one in the same, though.